experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -220,9 +226,29 @@ export enum ForestType {
 	Optimized = 1,
 }
 
+/**
+ * Selects which heuristics to use when encoding tree content.
+ * All encoding options here are compatible with the same decoder:
+ * the selection here does not impact compatibility.
+ * @alpha
+ */
+export enum TreeCompressionStrategy {
+	/**
+	 * Optimized for encoded size.
+	 * Use this in production to reduce bandwidth and storage use.
+	 */
+	Compressed = 0,
+	/**
+	 * Optimized for human readability.
+	 * Use this when debugging or testing and needing to inspect encoded tree content.
+	 */
+	Uncompressed = 1,
+}
+
 export const defaultSharedTreeOptions: Required<SharedTreeOptions> = {
 	jsonValidator: noopValidator,
 	forest: ForestType.Reference,
+	summaryEncodeType: TreeCompressionStrategy.Uncompressed,
```

Comment:
I think we want compressed by default so production use gets the compressed format. If that breaks too many things, leave a TODO here to change it later.